tests/hub-cloud/test_connect_instance.py

```diff
@@ -44,11 +45,8 @@ def test_connect_after_revoked_access():
             ln_setup.settings.instance.storage.root_as_str
             == "s3://lamindb-setup-private-bucket"
         )
-        delete_collaborator(
-            "laminlabs",
-            "static-test-instance-private-sqlite",
-            ln_setup.settings.user._uuid,
-            admin_hub,
+        collaborator_handler.delete_by_name(
```

Comment:
Can we turn this into `delete_by_slug`? 

So that the signature becomes the following?
```
"testuser2", "laminlabs/static-test-instance-private-sqlite"
```

Reply:
Done!

Reply:
Oh, thank you! That's a lot more intuitive now!